.github/workflows/mongo_backup.yaml

```diff
@@ -50,13 +56,15 @@ jobs:
 
           oc -n ${BUILD_NAMESPACE} start-build bc/${DB_BACKUP_APP} --wait
 
-  # deploy the backup image in Dev
-  deployDev:
-    name: Deploy DB_BACKUP_APP to Dev environment
+  # deploy the backup image
+  deploy:
+    if: ${{contains(['dev', 'test', 'prod'], github.event.inputs.environment)}}
```

Comment:
Isn't this already enforced by the options list?

Reply:
It actually does!

Reply:
Fixed 👍